src/libpatch/analysis/mod.rs

```diff
@@ -2,25 +2,19 @@ use std::io::{self, Write};
 use std::vec::Vec;
 
 use crate::modified_file::ModifiedFile;
-use crate::patch::{
-    FilePatchApplyReport,
-    TextFilePatch,
-    PatchDirection,
-};
+use crate::patch::{FilePatchApplyReport, PatchDirection, TextFilePatch};
 
 use std::fmt::Debug;
 
 mod multiapply;
 pub use self::multiapply::*;
 
-
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub enum NoteSeverity {
-    Warning
-    // TODO: Add more when needed
+    Warning, // TODO: Add more when needed
```

Comment:
This is definitely a change towards the worse. The *TODO* is not related to the `Warning` but to the enum as a whole.